src/_includes/layouts/home.liquid

```diff
@@ -1,86 +1,76 @@
 ---
 layout: layouts/base.liquid
 ---
-<section>
 
-  <div class="bg-parallax">
-    <div>
-      <img src="/assets/img/bg-triangle-gray.png" />
-    </div>
+<div class="relative overflow-clip">
+  <div class="absolute z-[-1]">
+    <img src="/assets/img/bg-triangle-gray.png" alt="Gray triangle"/>
   </div>
-
-  <div class="">
-    <div class="">
-      <div class="">
-        <h1 class="text-blue">{{ homeHeroRight.heroTitle }}</h1>
-        <h3 class="text-gray">{{ homeHeroRight.heroSubTitle }}</h3>
+  <div class="z-1 py-5 mx-8 desktop:mx-[8rem] bg-illustration bg-no-repeat h-[85vh]" style="background-image: url('/assets/img/illustration-gov.png'); background-size: contain;">
```

Comment:
Try to avoid inline `style` attributes.  I would move the `background-image` property into the external stylesheet.  The only time I would apply it inline is if the value was populated dynamically through frontmatter or something like that.  Use `bg-contain` class instead of setting the property in the `style` attribute.

Reply:
Yeah, I wasn't able to use a separate style sheet because of the way `webpack` is configured 

Currently, if I try to add an `image-url` to the `main.css` file I get a path error

```
ERROR in ./src/styles/main.css
Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
Error: Automatic publicPath is not supported in this browser
```


The current configuration allows `Webpack` to bundle the` main.js` file, which imports the `main.css` file and then processes CSS files using the specified loaders. It also includes the `MiniCssExtractPlugin` to extract CSS into separate files during the build process. The resulting output will include a bundled main.js and an extracted CSS file.

```
const path = require('path');
const MiniCssExtractPlugin = require('mini-css-extract-plugin');

module.exports = {
  entry: './src/scripts/main.js',
  output: {
    path: path.resolve(__dirname, '_site/styles'),
    filename: 'main.js'
  },
  plugins: [new MiniCssExtractPlugin()],
  module: {
    rules: [
      {
        test: /\.css$/,
        exclude: /node_modules/,
        use: [MiniCssExtractPlugin.loader, 'css-loader', 'postcss-loader']
      }
    ]
  }
};

```

Would you recommend a different or better approach for configuring webpack?

Reply:
Have you tried using the passthrough to copy the images?
`eleventyConfig.addPassthroughCopy("src/assets/img");`  

Also, in your stylesheet, use a relative path instead an absolute path from the web root.

Reply:
So, looking at the documentation and googling around, it doesn't appear easy to do this with 11ty.  Let me do some more research and testing to see if there's an easier way to do it than using short codes and css templates.

Reply:
It looks like we need to create a shortcode that will be used in the stylesheet, but I can't figure out how to use short codes in stylesheets with 11ty.

Reply:
To answer the first question, im using `eleventyConfig.addPassthroughCopy("src/assets/**");` Which uses a glob to capture anything in the assets folder and create a copy in the output `_site` folder. So the image is being passed through. However, I believe how `webpack` is configured it creates a styles folder with a `main.js` file with the modules and an external `main.css` file so im not sure where the path should come from should it be relative to the output folder or the source folder? if it's relative to the output folder, the path should be `..\assets\img\img.jpg` but I  Eleventy doesn't like that.

Reply:
interesting enough, when I add `publicPath: ''` to the `webpack.config.js` file about the `path` it doesn't throw an error and it places the images into the `styles` folder with a different file name which is hash?

Reply:
Right, the images are getting copied, but when the build process runs, it goes through the stylesheets and updates the paths of the images.  The hash is for cache busting (when the image changes, but the filename/path doesn't).  So, in order for the path to be correct, you need to be able to use short codes in the stylesheet. This way it gets updated with the appropriate path. https://github.com/11ty/eleventy-img/issues/46

Reply:
@tridnguyen Thanks for looking into this! Jerry is rolling off the project so I'll pick up the fix for the background images in a separate PR. Resolving this conversation for now.